ee/clickhouse/queries/paths/paths.py

```diff
@@ -217,3 +254,33 @@ def get_filtered_path_ordering(self) -> Tuple[str, str]:
                 "arraySlice(filtered_path, 1, %(event_in_session_limit)s)",
                 "arraySlice(filtered_timings, 1, %(event_in_session_limit)s)",
             )
+
+    def validate_results(self, results):
```

Comment:
I was thinking just work with this for now and not even put in the parameters for edge weighting until we decide exactly how we want that to work

Reply:
Hmm, if I don't get a chance to experiment with those settings, I wouldn't be confident if it's a good (or bad) solution.

I was hoping this is fine on the backend, since then I can 

1. create a possibly new FF to experiment with edge weights and see how they change things quickly on the UI, without any backend changes
2. If we decide to not go this route, easy enough to remove from the API, since no-one else would be using them.

Reply:
yeah good point